impala/README.md

```diff
@@ -0,0 +1,98 @@
+# Agent Check: Impala
+
+## Overview
+
+This check monitors [Impala][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+The Impala check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `impala.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Impala performance data. See the [sample impala.d/conf.yaml][4] for all available configuration options.
+
+Example with a single daemon:
+
+```yaml
+init_config:
+   
+instances:
+  ## @param service_type - string - required
+  ## The Impala service you want to monitor. Possible values are `daemon`, `statestore`, and `catalog`.
+  #
+- service_type: daemon
+
+  ## @param openmetrics_endpoint - string - required
+  ## The URL exposing metrics in the OpenMetrics format.
+  ##
+  ## The default port for the services are:
+  ## - Daemon: 25000
+  ## - Statestore: 25010
+  ## - Catalog: 25020
+  #
+  openmetrics_endpoint: http://%%host%%:25000/metrics_prometheus
+```
+
+You can also monitor several services at the same time with the same agent:
+
+```yaml
+init_config:
+   
+instances:
+
+- service_type: daemon
+  service: daemon-1
+  openmetrics_endpoint: http://<DAEMON-1-IP>:25000/metrics_prometheus
+- service_type: daemon
+  service: daemon-2
+  openmetrics_endpoint: http://<DAEMON-2-IP>:25000/metrics_prometheus
+- service_type: statestore
+  openmetrics_endpoint: http://<STATESTORE-IP>:25010/metrics_prometheus
+- service_type: catalog
+  openmetrics_endpoint: http://<CATALOG-IP>:25020/metrics_prometheus
+```
+
+2. [Restart the Agent][5].
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `impala` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][7] for a list of metrics provided by this integration.
+
+### Events
+
+The Impala integration does not include any events.
+
+### Service Checks
+
+See [service_checks.json][8] for a list of service checks provided by this integration.
+
+### Logs
+
+The Impala integration can collect logs from the Impala services and forward them to Datadog.
```

Comment:
```suggestion
The Impala integration can collect logs from the Impala services and forward them to Datadog. See the example configuration file on how to collect all logs
```
nit we should mention and link to the config file

Reply:
Added!